### DISCUSSION TITLE
Refused to load the script because it violates the following Content Security Policy directive

### DISCUSSION THREAD

**Discussion by luisrieke** [supabase/supabase#13987] [Questions]
I have a simple html/js website. I want to use Supabase for the first time to save when a checkbox is marked.

So I tried to add Supabase via head in the index.html:
`<head>[...]<script src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script></head>`

I tried to get Supabase via npm as explained here: [https://www.npmjs.com/package/@supabase/supabase-js](https://www.npmjs.com/package/@supabase/supabase-js)

But I get an error ❌

In the console it says:
`Refused to load the script 'https://cdn.jsdelivr.net/npm/@supabase/supabase-js' because it violates the following Content Security Policy directive: "script-src 'self' blob: filesystem:". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
`

I already tried to fix this with:
`<meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' cdn.jsdelivr.net blob: filesystem:">
`

But still does not work.

Can you help me?

**Comment by GaryAustin1** [comment]
```
<head>
    <meta charset="UTF-8">
    <title>Supabase Realtime Realworld</title>
    <script src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script>
</head>
```
Works for my pure js test site.

So it must be your environment, but you don't say what that is...

I suggest more googling on the web and include your environment.
I found many hits like this: https://stackoverflow.com/questions/34950009/chrome-extension-refused-to-load-the-script-because-it-violates-the-following-c
